sdk/servicebus/azure-servicebus/samples/async_samples/topic_subscription_with_rule_operations_async.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show selecting a message into Subscriptions on a Topic using various Filters asynchronously.
+"""
+
+# pylint: disable=C0111
+
+import os
+import time
+import asyncio
+from azure.core.exceptions import ResourceNotFoundError, ResourceExistsError
+from azure.servicebus.aio.management import ServiceBusAdministrationClient
+from azure.servicebus.management import (
+    TrueRuleFilter,
+    SqlRuleFilter,
+    SqlRuleAction,
+    CorrelationRuleFilter
+)
+from azure.servicebus.aio import ServiceBusClient
+from azure.servicebus import ServiceBusMessage
+
+CONNECTION_STR = os.environ['SERVICE_BUS_CONNECTION_STR']
+TOPIC_NAME = os.environ["SERVICE_BUS_TOPIC_NAME"]
+ALL_MSGS_SUBSCRIPTION_NAME = os.environ["SERVICE_BUS_ALL_MSGS_SUB_NAME"]
+SQL_FILTER_ONLY_SUBSCRIPTION_NAME = os.environ["SERVICE_BUS_SQL_FILTER_ONLY_SUB_NAME"]
+SQL_FILTER_WITH_ACTION_SUBSCRIPTION_NAME = os.environ["SERVICE_BUS_SQL_FILTER_ACTION_SUB_NAME"]
+CORRELATION_FILTER_SUBSCRIPTION_NAME = os.environ["SERVICE_BUS_CORR_FILTER_SUB_NAME"]
```

Comment:
I think those should be constant strings instead of os envs as we removed them from the `test-resources.json`, otherwise wouldn't we get error in the sample test running?